### PR TITLE
Update AboutUs.md

### PR DESCRIPTION
**Pull request by marcusngwenjian**
@CS2103AUG2016-W15-C3/documentation Ready for review

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -4,49 +4,88 @@ We are a team based in the [School of Computing, National University of Singapor
...
+* Be responsible in creating a comfortable workflow structure for the team
+<br>
 
-#### [Joshua Lee](http://github.com/lejolly)
-<img src="images/JoshuaLee.jpg" width="150"><br>
-Role: Developer <br>  
-Responsibilities: UI
+As a Code Quality Surveyor, I will
+* Look after the code quality
+* Ensure that they adhere to the Java coding standards
```
Comment (Mamtha3005) [inline docs/AboutUs.md]: State "the team" instead of "they....(adhere to the Java coding)..."

File: docs/AboutUs.md [hunk 2]
```diff
@@ -4,49 +4,88 @@ We are a team based in the [School of Computing, National University of Singapor
...
-Responsibilities: Data
+#### [Jazlyn Ang Chue Ching](https://github.com/turtle96)
+<img src="images/Jazlyn.png" width="150"><br>
+**Role**: Developer <br>
+**Responsibilities**: UI, Testing and Integrating of codes <br>
+
+As a Code Integrator, I will
+* Make sure code works well when integrating a new feature to system (by liaising with those in-charge of Documentation, Code Quality and Testing)
+* Make sure any new code follows software design principles for easier maintenance
+* Look after the Github repo
```
Comment (Mamtha3005) [inline docs/AboutUs.md]: "repository" instead of"repo"

File: docs/AboutUs.md [hunk 3]
```diff
@@ -4,49 +4,88 @@ We are a team based in the [School of Computing, National University of Singapor
...
 
-#### [Thien Nguyen](https://github.com/ndt93)
- Role: Developer <br>  
- Responsibilities: Threading
- 
- -----
+#### [Zuo ZhuoLin](https://github.com/ZuoZhuolin)
+<img src="images/ZhuoLin.png" width="150"><br>
+**Role**: Developer <br>
+**Responsibilities**: Dev Ops <br>
```
Comment (Mamtha3005) [inline docs/AboutUs.md]: Use full form of Dev Ops

File: docs/AboutUs.md [hunk 4]
```diff
@@ -4,49 +4,88 @@ We are a team based in the [School of Computing, National University of Singapor
...
+
+As a Scheduling Leader, I will
+* Set realistic deadline for different milestones of the project
+* Remind team mates when deadlines are nearing and help them to manage time
```
Comment (Mamtha3005) [inline docs/AboutUs.md]: Remove "and help them to manage time". Too general.

**Comment by marcusngwenjian** [comment]
Changes made.
@CS2103AUG2016-W15-C3/documentation Ready for review.

**Comment by Mamtha3005** [comment]
@CS2103AUG2016-W15-C3/integration ready for integration

**Pull request merged**